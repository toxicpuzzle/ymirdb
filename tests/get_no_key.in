SNAPSHOT
SET a 10
SNAPSHOT
GET a
CHECKOUT 1
GET a
CHECKOUT 2
GET a
bye
