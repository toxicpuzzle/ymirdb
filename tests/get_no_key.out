> saved as snapshot 1

> ok

> saved as snapshot 2

> [10]

> ok

> no such key

> ok

> [10]

> bye